amd64.env

```
@@ -1,6 +1,6 @@
 rosdistro=galactic
 rmw_implementation=rmw_cyclonedds_cpp
+base_image=ubuntu:22.04
```

Sorry, for galactic, it was 20.04.

```suggestion
base_image=ubuntu:20.04
```